packages/dx-react-demos/src/demo-sources/grid-filtering/disable-column-filtering.jsxt

```diff
@@ -0,0 +1,58 @@
+import React from 'react';<%&additionalImports%>
+import {
+  FilteringState,
+  IntegratedFiltering,
+} from '@devexpress/dx-react-grid';
+import {
+  Grid,
+  Table,
+  TableHeaderRow,
+  TableFilterRow,
+} from '@devexpress/dx-react-grid-<%&themeName%>';
+
+import {
+  generateRows,
+} from '../../../demo-data/generator';
+
+const filteringStateColumnExtensions = [
```

Comment:
What about const here? The columns config is inlined into the component's state.